src/item.cpp

```diff
@@ -4633,7 +4634,7 @@ bool item::reload( player &u, item_location loc, long qty )
         }
     }
 
-    if( ammo->charges == 0 ) {
+    if( ammo && ammo->charges == 0 ) {
```

Comment:
Yeah, thought about that first. Then thought that perhaps it was desirable to have one return point. I can just turn it into a return...